<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>732</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>547</width>
    <height>677</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>547</width>
    <height>732</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SADC Climate Forecasting Tool v1.3.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../cit/sadc.ico</normaloff>../cit/sadc.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>525</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>525</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>525</width>
      <height>50</height>
     </size>
    </property>
    <property name="title">
     <string>Output Directory</string>
    </property>
    <widget class="QPushButton" name="outputButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>93</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="outdirlabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>381</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>381</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>68</y>
      <width>224</width>
      <height>109</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>224</width>
      <height>109</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>224</width>
      <height>109</height>
     </size>
    </property>
    <property name="title">
     <string>Forecast Date</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Forecast Year:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fcstyearlineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="period1Radio">
       <property name="text">
        <string>Season</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="period2Radio">
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Period:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="periodComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>60</y>
      <width>294</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>294</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>294</width>
      <height>304</height>
     </size>
    </property>
    <property name="title">
     <string>Algorithms</string>
    </property>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>270</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>62</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Linear Regression</string>
     </property>
     <widget class="QCheckBox" name="LRcheckBox">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>20</y>
        <width>116</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>40</y>
        <width>116</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Correlation P-Value:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pvaluelineEdit">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>43</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="MLRcheckBox">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>20</y>
        <width>123</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MultiLinear</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>91</y>
       <width>270</width>
       <height>100</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>111</height>
      </size>
     </property>
     <property name="title">
      <string>Neural Networks</string>
     </property>
     <widget class="QCheckBox" name="NNcheckBox">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>28</y>
        <width>77</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="LSTMcheckBox">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>54</y>
        <width>77</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>LSTM</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="MLPcheckBox">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>54</y>
        <width>78</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>MLP</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="CNNcheckBox">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>54</y>
        <width>77</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>CNN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>80</y>
        <width>52</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Neurons:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="neuronslineEdit">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>80</y>
        <width>162</width>
        <height>19</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>200</y>
       <width>105</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stepwise P-Value:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="swpvaluelineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>137</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TrainingPeriod">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>184</y>
      <width>224</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>224</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>224</width>
      <height>100</height>
     </size>
    </property>
    <property name="title">
     <string>Training Period</string>
    </property>
    <widget class="QCheckBox" name="inclscorecheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Include in Statistics</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Start Year:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>End Year:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="startyearLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="endyearLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>291</y>
      <width>224</width>
      <height>354</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>224</width>
      <height>171</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>224</width>
      <height>500</height>
     </size>
    </property>
    <property name="title">
     <string>Predictors</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0" colspan="3">
      <widget class="QListWidget" name="predictorlistWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>81</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="addpredictButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="removepredictButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Predictor Months:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="predictMonthComboBox"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="predictMonth2ComboBox"/>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Aggregation Function:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="avgRadio_2">
       <property name="text">
        <string>Average</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QRadioButton" name="cumRadio_2">
       <property name="text">
        <string>Cumulation</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Min Lat:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Max Lat:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLineEdit" name="minlatLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="maxlatLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Min Lon:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Max Lon:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLineEdit" name="minlonLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLineEdit" name="maxlonLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>298</y>
      <width>294</width>
      <height>185</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>294</width>
      <height>185</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>294</width>
      <height>185</height>
     </size>
    </property>
    <property name="title">
     <string>Predictant Format</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Input Format:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="CSVRadio">
       <property name="text">
        <string>CSV</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QRadioButton" name="NetCDFRadio">
       <property name="text">
        <string>NetCDF</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Missing Values:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="missingvalueslineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Function:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QRadioButton" name="cumRadio">
       <property name="text">
        <string>Cumulation</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="avgRadio">
       <property name="text">
        <string>Average</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Zone Vector:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QPushButton" name="ZoneButton">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="zoneIDcomboBox">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QLabel" name="zonevectorlabel">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>271</width>
         <height>25</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>479</y>
      <width>294</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>224</width>
      <height>131</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>547</width>
      <height>131</height>
     </size>
    </property>
    <property name="title">
     <string>Predictant Input Data</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QListWidget" name="predictantlistWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>268</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>81</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="browsepredictantButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="clearpredictantButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>617</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>617</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="progresslabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>652</y>
      <width>531</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>531</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>531</width>
      <height>21</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
